bluebinder.c

```diff
@@ -764,6 +764,13 @@ bluebinder_callbacks_transact(
                         (code == ACL_DATA_RECEIVED) ? HCI_ACLDATA_PKT :
                         (code == SCO_DATA_RECEIVED) ? HCI_SCODATA_PKT : /* unreachable */ 0xFF;
 
+            if (packet[1] == 0x0e) {
```

Comment:
We can't enable this by default, it's device specific, we need to make it somehow configureable.

Reply:
I could imagine installable (device or platform specific) packet handling plugins, which would be conceptually very nice but at the same time it would make this thing so much more complicated, that I'm not sure if it's worth the trouble.